pset-iterators.js

```diff
@@ -43,6 +43,12 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents.find(function (president) {
+  return president.party === "Whig"
+})
+
+console.log(firstWhigPresident.party)
+
```

Comment:
@ChicoMangusta Good job here using the find method to return the first president who belongs to the `Whig` party. One suggestion when you `console.log(firstWhigPresident.party)` you can leave off the `.party` and just `console.log(firstWhigPresident)`

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)